tests/periph_ptp_clock/main.c

```diff
@@ -77,7 +77,7 @@ static int test_speed_adjustment(const int16_t speed)
     print_str(" (~");
     {
         int64_t tmp = speed * 100000ULL;
-        tmp = (tmp + UINT16_MAX / 2) / UINT16_MAX;
+        tmp /= UINT32_MAX;
```

Comment:
I dropped scientific rounding here. I don't think this is really needed for the debug message and makes the code harder to read.